sympy/utilities/lambdify.py

```diff
@@ -88,22 +90,74 @@
     "Matrix": "array",
     "MutableDenseMatrix": "array",
     "ImmutableMatrix": "array",
+    "ImmutableDenseMatrix": "array"
+
+}
+
+# most of the functions of scipy is similar to numpy. Checked each functions.
+SCIPY_TRANSLATIONS = {
+    "acos": "arccos",
+    "acosh": "arccosh",
+    "arg": "angle",
+    "asin": "arcsin",
+    "asinh": "arcsinh",
+    "atan": "arctan",
+    "atan2": "arctan2",
+    "atanh": "arctanh",
+    "ceiling": "ceil",
+    "E": "e",
+    "im": "imag",
+    "ln": "log",
+    "Max": "amax",
+    "Min": "amin",
+    "Mod": "mod",
+    "oo": "inf",
+    "re": "real",
+    "SparseMatrix": "array",
```

Comment:
I am not sure about this(I searched in internet, but didn't find) , but because array is used in Scipy and numpy very similarly,so I thought we can map them.